File: CS/Q205815/Form1.cs
using System;
using System.Data;
using System.Windows.Forms;

namespace Q205815
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            myGridControl1.DataSource = GetPersonDataTable();

        }
        DataTable GetPersonDataTable()
        {
            DataTable table = new DataTable();
            table.TableName = "Persons";
            table.Columns.Add(new DataColumn("FirstName", typeof(string)));
            table.Columns.Add(new DataColumn("SecondName", typeof(string)));
            table.Columns.Add(new DataColumn("Age", typeof(int)));
            table.Columns.Add(new DataColumn("ID", typeof(int)));
            for (int i = 0; i < 150; i++)
                table.Rows.Add("FirstName " + i, "SecondName " + i, 20 + i / 15, i);
            return table;
        }
        private void OnShowPrintPreviewSimpleButtonClick(object sender, EventArgs e)
        {
            myGridControl1.ShowPrintPreview();
        }
    }
}

File: VB/Q205815/Form1.vb
Imports System
Imports System.Data
Imports System.Windows.Forms

Partial Public Class Form1
    Inherits Form

    Public Sub New()
        InitializeComponent()
    End Sub

    Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
        myGridControl1.DataSource = GetPersonDataTable()
    End Sub

    Private Function GetPersonDataTable() As DataTable
        Dim table As DataTable = New DataTable()
        table.TableName = "Persons"
        table.Columns.Add(New DataColumn("FirstName", GetType(String)))
        table.Columns.Add(New DataColumn("SecondName", GetType(String)))
        table.Columns.Add(New DataColumn("Age", GetType(Integer)))
        table.Columns.Add(New DataColumn("ID", GetType(Integer)))

        For i As Integer = 0 To 150 - 1
            table.Rows.Add("FirstName " & i, "SecondName " & i, 20 + i \ 15, i)
        Next

        Return table
    End Function

    Private Sub OnShowPrintPreviewSimpleButtonClick(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
        myGridControl1.ShowPrintPreview()
    End Sub
End Class